tests/dbus/_misc.py

```diff
@@ -65,9 +69,16 @@ class ServiceR(ServiceABC):
     """
     Handle starting and stopping the Rust service.
     """
+    def __init__(self, not_sim=False):
+        super(ServiceR, self).__init__()
+        self._not_sim = not_sim
 
     def setUp(self):
-        self._stratisd = subprocess.Popen([os.path.join(_STRATISD), '--sim'])
+        if MODE == SIMULATOR and not self._not_sim:
+            self._stratisd = subprocess.Popen([os.path.join(_STRATISD), '--sim'])
```

Comment:
This setup seems to be erring on the side of caution, which is good. I routinely run these tests on my personal machine, and I wouldn't welcome a very nasty surprise.